.github/workflows/ci.yml

```diff
@@ -65,8 +65,10 @@ jobs:
           if [[ "${{ matrix.tags }}" == "test_motus" ]]; then
             wget https://raw.githubusercontent.com/motu-tool/mOTUs/master/motus/downloadDB.py
             python downloadDB.py --no-download-progress
-            echo 'tool,db_name,db_params,db_path' > 'database_motus.csv'
-            echo "motus,db_mOTU,,db_mOTU" >> 'database_motus.csv'
+            echo 'tool,db_name,db_params,db_type,db_path' > 'database_motus.csv'
+            echo "motus,db1_mOTU,,short,db_mOTU" >> 'database_motus.csv'
+            echo "motus,db2_mOTU,prep_long,long,db_mOTU" >> 'database_motus.csv'
```

Comment:
Is `prep_long` a valid argument to mOTUs?

Reply:
I am actually not so sure. But I don't get any errors with `pre_long`. 
The example from the document is: `motus prep_long -i long_reads.fasta -o converted_long_reads.fasta.gz`

Reply:
I deleted `pre_long` since I am not certain the usage of it.